app/hooks/project.ts

```diff
@@ -1,12 +1,18 @@
 import { useContract, useStarknetCall } from '@starknet-react/core';
 import type { Abi } from 'starknet';
 
-import ProjectAbi from '../abi/CarbonableProject_abi.json';
+import ProjectAbiTestnet from '../abi/testnet2/CarbonableProject_abi.json';
```

Comment:
It's strange to have `../abi/testnet2/CarbonableProject_abi.json` for the testnet-1 ?
Same for below import